web-animations-1/Overview.bs

```diff
@@ -4810,8 +4827,15 @@ interface KeyframeEffect : AnimationEffect {
     This is `null` if the [=effect target=] is an {{Element}} or is itself `null`.
     When not `null`, this must be a [=pseudo-element=] selector with
     [[selectors-4#pseudo-element-syntax|valid syntax]].
-    This will not accept invalid values and will always return the selector in
-    its canonical form (i.e. `::before` instead of `:before`).
+
+    This will only accept `null` or syntactically-valid selectors
+    (i.e. `:before`, `:after`, `:first-letter`, `:first-line`, or a string starting with `::`).
+    If such a string is not given, this will throw a {{DOMException}} with
+    error name {{SyntaxError}}.
+
+    If one of the legacy single-colon selectors is given, the selector is
```

Comment:
This needs to be above the 'If such a string' line, otherwise we throw the exception before ever getting here.